index.js

```diff
@@ -1,8 +1,9 @@
-/* jshint node: true */
+/* eslint-env node */
+
 'use strict';
 
 var RSVP = require('rsvp');
-var fs = require('fs');
+var fs = require('fs-extra');
```

Comment:
how come? `fs` is built in to node so afaik there's no need to add it explicitly.